### PR TITLE
Increase kafka timeout and add documentation

### PR DESCRIPTION
**Pull request by Y4Rv1K**
Increase time to wait kafka scheduler to 1800 sec
Add kafka.md to docs with short kafka usage guide

IssueID: Internal

### CODE REVIEW

**Comment by Y4Rv1K** [comment]
@OleksandrBerchenko please review kafka timeout increase

File: docs/kafka.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+# Kafka on Mesos Guide
+Short guide about kafka-mesos and kafka usage at MI cluster
+
+## kafka-mesos.sh
+
+Kafka-mesos.sh utility built to control kafka brokers/server on top of mesos.
```
Comment (OleksandrBerchenko) [inline docs/kafka.md]: is built

File: docs/kafka.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        resources: cpus:1.00, mem:2048, heap:1024
+        failover: delay:10s, max-delay:60s
+      
+        id: 2
+        active: false
+        state: stopped
+        resources: cpus:1.00, mem:2048, heap:1024
+        failover: delay:10s, max-delay:60s
+
+Now you be able to start specific broker:
```
Comment (OleksandrBerchenko) [inline docs/kafka.md]: are able

Comment (OleksandrBerchenko) [inline docs/kafka.md]: a specific

File: docs/kafka.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+        kafka-console-consumer.sh --zookeeper zookeeper.service.consul:2181 --topic test --from-beginning
+
+    You should obtain:
+
+        message one
+        message two
+
+
+_Note: If you have each of the above commands (producer and consumer) running in a different terminal then you should be able to type messages into the producer terminal and see them appear in the consumer terminal._
```
Comment (OleksandrBerchenko) [inline docs/kafka.md]: and observe them

**Comment by Y4Rv1K** [comment]
fixed

**Comment by Y4Rv1K** [comment]
@OleksandrBerchenko please review and merge

**Comment by OleksandrBerchenko** [comment]
@Y4Rv1K, done.

**Pull request merged**